test/components/link-community/link-community.vdiff.js

```diff
@@ -0,0 +1,37 @@
+
+import '../../../src/components/community/link-community.js';
+import '../../../src/components/community/button-community.js';
+import { expect, fixture, html } from '@brightspace-ui/testing';
+
+const opts = { viewport: { height: 450 } };
+
+const langArticleMap = {
+	'en': 4545,
+	'ar-SA': 10489,
+	'es-MX': 9214,
+	'pt-BR': 9978,
+	'nl-NL': 7833,
+	'fr-CA': 6739
+};
```

Comment:
Can you `export` this from somewhere common and share it between all your tests and the demo?

Reply:
Add this to the list of language maps 😄